src/api/app/views/webui/package/_files_view.html.haml

```diff
@@ -23,6 +23,7 @@
       = link_to(new_project_package_files_path(project, package)) do
         %i.fas.fa-plus-circle.text-primary
         Add File
+    = "Checkout this package with osc -A {FIXME:api_url} checkout #{project}/#{package} && cd $_"
```

Comment:
There is `Configuration.api_url` but that one is not necessarily set. You could display this only if it's set.

Reply:
OK. did that.